qipr_approver/approver/workflows/project_crud.py

```diff
@@ -144,4 +146,96 @@ def current_user_can_perform_project_delete(current_user,project):
     if (toast_message == "" and project.get_is_editable() is not True):
         return 'You are not allowed to delete/edit this project.'
     project.delete(current_user)
-    return 'Deleted Project'
\ No newline at end of file
+    return 'Deleted Project'
+
+def get_all_projects():
+    """
+    This returns a list of all the existing projects
+    """
+    return Project.objects.all()
+
+def get_similar_projects(project):
+    projects = get_all_projects()
```

Comment:
We don't care about all projects. We only care about ones that have already been approved.